Main/develop

nishanthprakash opened the pull request.

franzliedke commented:
This has a lot of strange indentation changes.

What exactly does the resizer change do?

nishanthprakash commented:
This resizer change sets up the resizer's initial position, i.e. at the edge of the first video stream.

jprberlin commented:
Please check indentation

nishanthprakash commented:
My previous commit had a lot of bad indentation which I corrected with the 'resizer init' commit (this new 'resizer init' commit had an unintended indentation change which I reverted in my indentation commit) -- sorry about this. So the indentations seem in place now. Is there a particular indentation issue you want me to look into and revise?

franzliedke commented on video-player.html:
```diff
@@ -224,7 +223,9 @@
-        // manage bindings
+        document.querySelector("#resizer").style.left = this.videoStreamList[0].style.width;
+
+	// manage bindings
```
This is off

franzliedke commented on video-player.html:
```diff
@@ -305,39 +306,36 @@
         var numVideos = this.videoStreamList.length;
         // --------------------------------------------
 
+	document.querySelector("#resizer").style.left = this.videoStreamList[0].style.width;
```
This is off.

franzliedke commented on video-player.html:
```diff
@@ -305,39 +306,36 @@
-                  (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )) {
+          if ((height(this.videoStreamList[0]) <= height(this.videoStreamList[1]))&&
+                   (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
+                   (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )){
```
Please don't remove the spaces around the `&&` and before the opening `{`.

franzliedke commented on video-player.html:
```diff
@@ -305,39 +306,36 @@
           this.videoStreamList.slice(-1)[0].resize(this.videoStreamList.length);
-	
-	this.videoStreamList[1].resize(this.videoStreamList.length);
+	  this.videoStreamList[1].resize(this.videoStreamList.length);
```
This is off.

franzliedke commented on video-player.html:
```diff
@@ -305,39 +306,36 @@
-                  (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )) {
+          if ((height(this.videoStreamList[0]) >= height(this.videoStreamList[1]))&&
+                   (document.documentElement.clientHeight <= computedHeight(this) + controlbarHeight ) ||
+                   (document.documentElement.clientHeight > computedHeight(this) + controlbarHeight )){
```
Same here.

franzliedke commented on video-player.html:
```diff
@@ -305,39 +306,36 @@
           this.videoStreamList.slice(-1)[0].resize(this.videoStreamList.length);
           this.videoStreamList[0].resize(this.videoStreamList.length);
-       	this.videoStreamList.slice(-1)[1].resize(this.videoStreamList.length);
+	  this.videoStreamList.slice(-1)[1].resize(this.videoStreamList.length);
```
There's also a space missing here.

franzliedke commented:
I commented on a few. ;)

Which editor are you using? If compatible, I would suggest you install an [EditorConfig plugin](http://editorconfig.org/), I will push an appropriate config file soon.

nishanthprakash commented:
Thanks Franz. I use vi. The indentation seems appropriate as can be seen here: https://github.com/nishanthprakash/HTML5Player/blob/main/develop/video-player.html. 

I am not sure why the diffs are showing up such bad indentation.

franzliedke commented:
Hmm, not sure what is going on, but I can definitely see these faulty changes in the [patch file for this pull request](https://patch-diff.githubusercontent.com/raw/openHPI/HTML5Player/pull/16.patch).

nishanthprakash commented:
Maybe my definitions of "1 tab=8 spaces" differs from definition on other systems? I will just use spaces and see if this is resolved.

nishanthprakash commented:
yup, tabs were the issue.
